gearcmd/worker.go

```diff
@@ -40,11 +40,17 @@ func (conf TaskConfig) Process(job baseworker.Job) ([]byte, error) {
 	}
 	if err != nil {
 		data["error_message"] = err.Error()
-		log.Printf(kayvee.FormatLog("gearcmd", "error", "END_WITH_ERROR", data))
+		data["type"] = "gauge"
```

Comment:
I was going to move it up, since it is repeated, but keeping it close to the value being set made it feel like it would make that piece of code more readable (as in, the `type:gauge` provides context to the `value` field)